airflow/cli/cli_parser.py

```diff
@@ -778,6 +782,12 @@ class GroupCommand(NamedTuple):
         func=lazy_load_command('airflow.cli.commands.dag_command.dag_list_dags'),
         args=(ARG_SUBDIR, ARG_OUTPUT),
     ),
+    ActionCommand(
+        name='generate_kubernetes_pod_yaml',
```

Comment:
```suggestion
        name='generate-kubernetes-pod-yaml',
```
Commands should use dash as a separator. See: https://github.com/apache/airflow/pull/10669
I forgot one command yet, but I will correct it in a moment.

Reply:
Good point

Reply:
Actually looks like "next_execution" might need changing as well. We definitely need to decide dash or underscore

Reply:
PR: https://github.com/apache/airflow/pull/10720